src/data/thunks.js

```diff
@@ -284,8 +284,9 @@ export function pollAttempt(url) {
       dispatch(setActiveAttempt({
         activeAttempt: updatedAttempt,
       }));
-      if (data.status === ExamStatus.SUBMITTED) {
+      if ([ExamStatus.SUBMITTED, ExamStatus.ERROR].includes(data.status)) {
         dispatch(expireExamAttempt());
+        updateAttemptAfter(exam.course_id, exam.content_id)(dispatch);
```

Comment:
I'm not sure I understand this call, shouldn't this be dispatched as:
```javascript
dispatch(updateAttemptAfter(exam.course_id, exam.content_id))
```
like the rest of the actions?

Reply:
this thing is a bit weird, it gets passed dispatch since this thunk calls to run another thunk passed to it as a third argument and than 'update the exam after'. We just need to do the after part so no third argument here. You can see the other uses of this throughout the file as an example tho.

Reply:
My bad. I was not used to this way of using the thunk.